### DISCUSSION TITLE
Binding free energy calculation using MDtraj output (from NAMD)

### DISCUSSION THREAD

**Discussion by ShubhaBarik** [mdtraj/mdtraj#1869] [Q&A]
Hello!
How to calculate **binding free energy** between protein and ligand (system size= 100,000+ atoms) **after MD simulation**? Used NAMD for simulations? Thanks in advance.

**Comment by mattwthompson** [comment]
Binding free energy isn't something you can get out of a single MD run, it takes a certain amount of setup. There are a bunch of different ways to do this, I'm not an expert but here are some references that can point you in the right direction:

* https://livecomsjournal.org/index.php/livecoms/article/view/v2i1e18378
* https://openfree.energy/

I've never used NAMD so I don't know if there's a preferred tool used in that community.
